source/import-font-and-images-assets/index.html.md.erb

```diff
@@ -12,28 +12,30 @@ To use the font and image assets from GOV.UK Frontend, you can either:
 
 ## Serve the assets from the GOV.UK Frontend assets folder - recommended
 
-Set up your routing so requests for files in `<YOUR-SITE-URL>/assets` are served from `/node_modules/govuk-frontend/dist/govuk/assets`.
+<%= warning_text('The file paths on this page point to asset folders that were updated in GOV.UK Frontend v5.10.0') %>
```

Comment:
I think HTML should work here

```suggestion
<%= warning_text('The file paths on this page point to asset folders that were updated in <a href="https://github.com/alphagov/govuk-frontend/releases/tag/v5.10.0" GOV.UK Frontend v5.10.0<a/>') %>
```

Reply:
Thanks @domoscargin! In that case, could we just put a blanket message in this inset text?

If I go with committing this, I'll revert all the other file path changes. But I can still add those specific file path for the assets.

```suggestion
<%= warning_text('If you're using the refreshed GOV.UK  branding in your service, change the file paths on this page from `.../assets/` to `...assets/rebrand/`. See release notes for <a href="https://github.com/alphagov/govuk-frontend/releases/tag/v5.10.0" GOV.UK Frontend v5.10.0<a/>') %>.') %>
```